.travis.yml

```diff
@@ -1,6 +1,5 @@
 matrix:
     include:
-        - dist: xenial
```

Comment:
Doesn't have clang-9 in repo, and it uses the same kernel anyway (see #229). If we think it's valuable I can use apt.llvm.org